articles/active-directory/develop/quickstarts/active-directory-android.md

```diff
@@ -0,0 +1,180 @@
+> 1. Enter a name for your application and click **Register**.
+> 1. Follow the instructions to download and automatically configure your new application for you in one click.
+> [!div renderon="docs"]
+> ### Option 2: Register and manually configure your application and code sample
+> #### Step 1: Register your application
+> 
+> 1. To register an application, go to the [Azure portal - Application Registration [Prod]](https://aka.ms/registeredappsprod) and select **New registration**.
+> 1. Enter a name for your application and click **Register**.
+> 1. Select the **Authentication** page, then add `msal{AppId}://auth` (where *{AppId}* is the application ID from the application you just registered), select **Installed client** under **Type**, and then select **Save**.
+>
+>> [!TIP]
+>> To know the *Application ID*, go to Overview page.
+
+> [!div renderon="portal" class="sxs-lookup"]
+> ### Step 1: Configure your application
+> For the code sample for this quickstart to work, you need to add a reply URL as **msal{AppId}://auth** (where {AppId} is the application ID of your app).
+> > [!div renderon="portal" id="makechanges" class="nextstepaction"]
+> > [Make this change for me]()
+>
+> > [!div id="appconfigured" class="hidden"]
+> > ![Already configured](../../../../includes/media/active-directory-develop-common/green-check.png) Your application is configured with these attributes
+
+#### Step 2: Download your web server or project
+
+* [Download the Android Studio Project](https://github.com/Azure-Samples/active-directory-android-native-v2/archive/master.zip)
+
+#### Step 3: Configure your project
+
+1. Extract and open the project in Android Studio.
+1. Under **app** > **java** > **<i>{host}.{namespace}</i>**, open **MainActivity**.
+1. Replace the line starting with `final static String CLIENT_ID` with the following code:
+
+    ```java
+    final static String CLIENT_ID = "[Enter the application Id here]";
+    ```
+1. Open: **app** > **manifests** > **AndroidManifest.xml**.
+1. Add the following activity to the **manifest\application** node. This code snippet registers a **BrowserTabActivity** to allow the OS to resume your application after completing the authentication:
+
+    ```xml
+    <!--Intent filter to capture System Browser calling back to our app after Sign In-->
+    <activity
+        android:name="com.microsoft.identity.client.BrowserTabActivity">
+        <intent-filter>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+    
+            <!--Add in your scheme/host from registered redirect URI-->
+            <!--By default, the scheme should be similar to 'msal[appId]' -->
+            <data android:scheme="msalEnter_the_Application_Id_here"
```

Comment:
Change `msalEnter_the_Application_Id_here` to `msalenter_the_application_id_here` as Manifest is complaining about case sensitivity

Reply:
@kreedula - A note is that I sync'ed with Daniel Dobalian who will propose a change in the code sample to better accommodate this situation. He will likely contact you soon about it - let me know when the change in code sample is done so I can update this content

Reply:
Thanks @andretms LGTM